src/einsteinpy/hypersurface/schwarzschildembedding.py

```diff
@@ -0,0 +1,193 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy import units as u
+from mpl_toolkits import mplot3d
+
+
+class SchwarzschildEmbedding:
+    def __init__(self, M):
+        """
+      Constructor
+      Initialize mass and embedding initial radial coordinate in appropiate units
+      in order to render the plots of the surface in finite grid. The initial r
+      is taken to be just greater than schwarzschild radius but it is important
+      to note that the embedding breaks at r < 9m/4.
+
+      Parameters
+      ----------
+      M : ~astropy.units.kg
```

Comment:
`__init__` has only `M` as parameter but docstring says otherwise. Parameters & docstring should match.